### ISSUE TITLE
Class name of tempalted type wrong for re-instantiation.

### ISSUE THREAD

**Issue by jviereck** [stack-of-tasks/sot-core#71]
The following code:

``` python
filter0 = FIRFilter_Vector_double("ctrl_joint_torques_filter")
filter1 = FIRFilter_Vector_double("ctrl_joint_torques_filter") # This should return `filter0` again
```

creates the exception:

```
error: Found an object named ctrl_joint_torques_filter,
but this object is of type FIRFilter<Vector,double> and not FIRFilter_Vector_double
```

**Comment by jmirabel** [comment]
I believe this is easy to fix. The two following lines should generate the same string:
https://github.com/stack-of-tasks/sot-core/blob/33af0358bdd9f2fd0577b47fdd4c6373e1091f95/src/matrix/fir-filter.cpp#L35
and
https://github.com/stack-of-tasks/sot-core/blob/33af0358bdd9f2fd0577b47fdd4c6373e1091f95/src/matrix/fir-filter.cpp#L46

**Comment by olivier-stasse** [comment]
Fix by 4c5d21c
